### PR TITLE
fix: Grammar/font fixes

### PR DESCRIPTION
**Pull request by abhilash-kumar-nair**

### CODE REVIEW

**Comment by abhilash-kumar-nair** [comment]
@efredriksson-modelon , @jacobtaxen , Fixes some of the issues reported - See the updated docs here - https://modelon-impact-client.readthedocs.io/en/minor-doc-fixes/index.html

File: docs/source/setup.rst [hunk 1]
```diff
@@ -80,14 +80,14 @@ When the key has been entered the first time, it will be stored in the credentia
 to be used in future requests, and the prompt will not be shown again. If the key is
 changed, the file may be deleted which will cause the prompt to be shown again.
 Alternatively, the credentials file may be updated by specifying the ``api_key``
-argument in interactive mode:
+argument in an interactive mode:
```
Comment (jacobtaxen) [inline docs/source/setup.rst:83]: I think "in interactive mode" or "in the interactive mode" fits better here

File: docs/source/tutorial.rst [hunk 2]
```diff
@@ -43,14 +43,14 @@ We could fetch a list of available analysis methods and print them by executing:
```
Comment (jacobtaxen) [inline docs/source/tutorial.rst:43]: While you're at it, could you remove the comma in "There are different analysis methods, one can choose from" on line 36?

**Review by jacobtaxen — CHANGES_REQUESTED** [review]
Very good, it's so satisfying to see spelling errors being removed

**Review by jacobtaxen — APPROVED** [review]

**Pull request merged**